newsroom/agenda/email.py

```diff
@@ -84,7 +85,7 @@ def send_coverage_request_email(user, message, item):
     email = user.get("email")
 
     # send coverage request email copy to current User.
-    recipients.append(email)
+    cc = [email] if app.config.get("INCLUDE_CURRENT_USER_IN_CC") else []
```

Comment:
would add something to the config name to make it more obvious it's related to coverage request, eg.:
`COVERAGE_REQUEST_EMAIL_CC`